pkg/operator/ceph/operator.go

```diff
@@ -199,6 +199,10 @@ func (o *Operator) startSystemDaemons(clusterSpec *cephv1.ClusterSpec) error {
 		return errors.Wrapf(err, "invalid csi params")
 	}
 
+	if err = csi.ValidateCSIVersion(o.context.Clientset, o.operatorNamespace, csi.CSIParam.CSIPluginImage, o.rookImage, o.securityAccount); err != nil {
+		return errors.Wrapf(err, "invalid csi version: %v", err)
```

Comment:
`Wrapf` already wraps the error, so there is no need to include `%v` with `err`

Reply:
Ah. You are correct!